xml/ha_config_cli.xml

```diff
@@ -356,9 +387,187 @@ Operations' defaults (advisory minimum):
+          short description, and a procedure. Each procedure is divided
+          into a series of steps, performed in the given order. </para>
+      <para>Each step contains a list of required and optional parameters,
+        along with a short description and its default value.</para>
+
+      <para>Each cluster script understands a set of common parameters.
+        These parameters can be passed to any script:</para>
+
+      <table>
+        <title>Common Parameters</title>
+        <tgroup cols="3">
+          <thead>
+            <row>
+              <entry>Parameter</entry>
+              <entry>Argument</entry>
+              <entry>Description</entry>
+            </row>
+          </thead>
+          <tbody>
+            <row>
+              <entry><parameter>action</parameter></entry>
+              <entry><replaceable>INDEX</replaceable></entry>
+              <entry>If set, only execute a single action (index, as
+                returned by verify)</entry>
+            </row>
+            <row>
+              <entry><parameter>dry_run</parameter></entry>
+              <entry><replaceable>BOOL</replaceable></entry>
+              <entry>If set, simulate execution only (default: no) </entry>
+            </row>
+            <row>
+              <entry><parameter>nodes</parameter></entry>
+              <entry><replaceable>LIST</replaceable></entry>
+              <entry>List of nodes to execute the script for</entry>
+            </row>
+            <row>
+              <entry><parameter>port</parameter></entry>
+              <entry><replaceable>NUMBER</replaceable></entry>
+              <entry>Port to connect on</entry>
+            </row>
+            <row>
+              <entry><parameter>statefile</parameter></entry>
+              <entry><replaceable>FILE</replaceable></entry>
+              <entry>When single-stepping, the state is saved in the given
+                file </entry>
+            </row>
+            <row>
+              <entry><parameter>sudo</parameter></entry>
+              <entry><replaceable>BOOL</replaceable></entry>
+              <entry>If set, crm will prompt for a sudo password and use sudo
+                when appropriate (default: no) </entry>
+            </row>
+            <row>
+              <entry><parameter>timeout</parameter></entry>
+              <entry><replaceable>NUMBER</replaceable></entry>
+              <entry>Execution timeout in seconds (default: 600) </entry>
+            </row>
+            <row>
+              <entry><parameter>user</parameter></entry>
+              <entry><replaceable>USER</replaceable></entry>
+              <entry>Run script as the given user </entry>
+            </row>
+          </tbody>
+        </tgroup>
+      </table>
+    </sect3>
+
+    <sect3 xml:id="sec.ha.manual_config.clusterscripts.verify_run">
+      <title>Verifying and Running a Cluster Script</title>
+      <para>Before you actually run a cluster script it is advisable to
```

Comment:
Perhaps shorten to: Before running a cluster script review the actions that it will perform and verify its parameters to avoid problems.